cli/cmd/sync2.go

```diff
@@ -29,13 +30,15 @@ func syncConnectionV2(ctx context.Context, cqDir string, sourceClient *clients.S
 	}
 	defer destClients.Close()
 
-	tables, err := sourceClient.GetTables(ctx)
+	tables, err := sourceClient.GetTablesForSpec(ctx, &sourceSpec)
 	if err != nil {
 		return fmt.Errorf("failed to get tables for source %s: %w", sourceSpec.Name, err)
 	}
 
+	tableCount := len(schema.Tables(tables).FlattenTables())
+	fmt.Printf("Source %s will sync %d tables.\n", sourceSpec.Name, tableCount)
 	if !noMigrate {
-		fmt.Println("Starting migration for:", sourceSpec.Name, "->", sourceSpec.Destinations)
+		fmt.Printf("Starting migration for: %s -> %s\n", sourceSpec.Name, sourceSpec.Destinations)
```

Comment:
Just `\n`? Windows CLI may not look good

Reply:
I can revert this since I ended up not really changing this line anyway, but I believe Go's implementation of `Println` also just appends `\n`, so we'd be in good company: https://github.com/golang/go/blob/master/src/fmt/print.go#L1225